sdk/messaging/azservicebus/internal/atom/entity_manager.go

```diff
@@ -419,31 +419,15 @@ func TraceReqAndResponseMiddleware() MiddlewareFunc {
 	}
 }
 
-type feedEmptyError struct {
-	response *http.Response
-}
+type feedEmptyError struct{}
 
-func (e feedEmptyError) RawResponse() *http.Response {
-	return e.response
-}
 func (e feedEmptyError) Error() string {
 	return "entity does not exist"
 }
 
-func NotFound(err error) (bool, *http.Response) {
+func IsFeedEmptyError(err error) bool {
 	var feedEmptyError feedEmptyError
-
-	if errors.As(err, &feedEmptyError) {
-		return true, feedEmptyError.RawResponse()
-	}
-
-	return false, nil
-}
-
-func isEmptyFeed(b []byte) bool {
-	var emptyFeed QueueFeed
-	feedErr := xml.Unmarshal(b, &emptyFeed)
-	return feedErr == nil && emptyFeed.Title == "Publicly Listed Services"
+	return errors.As(err, &feedEmptyError)
```

Comment:
Nit:  you could define `feedEmptyError` to be a sentinel error.  Then you can use `errors.Is()` and not need a local for capturing.

Reply:
Great idea, switched it over to a sentinel error.